relayer/pkg/chainlink/keys/utils.go

```diff
@@ -28,7 +28,7 @@ func PubKeyToContract(pubkey starksig.PublicKey, classHash, salt *big.Int) []byt
 		pedersen.ArrayDigest(pubkey.X),
 	)
 
-	// pad big.Int to 32 bytes if needed
+	// pad big.Int to 32 bytes if needed:w
```

Comment:
```suggestion
	// pad big.Int to 32 bytes if needed
```

Reply:
ah dang it...still occasionally forgetting to hit `esc` before exiting terminal based editors lol